module/Activity/src/Activity/Service/Activity.php

```diff
@@ -132,7 +133,7 @@ public function createUpdateProposal(ActivityModel $oldActivity, array $params,
             $em->remove($oldUpdate);
             $em->flush();
 
-            return $oldProposal;
+            return false;
```

Comment:
I think this only would be an issue if an activities status would change from accepted to rejected, which rarely (never?) occurs.